pymc3/tests/test_distributions_random.py

```diff
@@ -498,6 +492,33 @@ class TestMoyal(BaseTestDistribution):
     ]
 
 
+class TestSkewNormal(BaseTestDistribution):
+    pymc_dist = pm.SkewNormal
+    pymc_dist_params = {"mu": 0.0, "sigma": 1.0, "alpha": 5.0}
+    expected_rv_op_params = {"mu": 0.0, "sigma": 1.0, "alpha": 5.0}
+    reference_dist_params = {"loc": 0.0, "scale": 1.0, "a": 5.0}
+    reference_dist = seeded_scipy_distribution_builder("skewnorm")
+    tests_to_run = [
+        "check_pymc_params_match_rv_op",
+        "check_pymc_draws_match_reference",
+        "check_rv_size",
+    ]
+
+
+class TestRice(BaseTestDistribution):
```

Comment:
We should add an additional test with the `b` parametrization. There are some examples already of testing alternative parametrizations. For that the `params_match_rv_op` test is enough.

Reply:
Sure, I am going to follow the Normal distribution test for tau and sd.